test/insertion_order_parser_test.dart

```diff
@@ -280,12 +274,12 @@ void main() {
               InsertionOrder_Budget_BudgetUnit.BUDGET_UNIT_UNSPECIFIED
           ..automationType = InsertionOrder_Budget_InsertionOrderAutomationType
               .INSERTION_ORDER_AUTOMATION_TYPE_NONE
-          ..budgetSegments.add(segment);
+          ..activeBudgetSegment = segment;
         final expected = insertionOrderTemplate..budget = budget;
         expect(actual, expected);
       });
 
-      test('multiple budgetSegments', () {
+      test('multiple budgetSegments that contains one active', () {
```

Comment:
good test

Reply:
do you have one for an unhappy path e.g. no budget segments?

Reply:
yep its [here](https://github.com/googleinterns/dv360-excel-plugin/pull/50/files/df81e8dbf19a44d5365def478d7a52e807efb808#diff-c51d0475621a2ed583a9e1278eb92a14L235-L255)